backend/src/Controllers/voterRolls.controller.ts

```diff
@@ -69,13 +84,78 @@ const editElectionRoll = async (req: any, res: any, next: any) => {
     }
 }
 
+const changeElectionRollState = async (req: any, res: any, next: any) => {
+    console.log(`-> electionRolls.changeElectionRollState ${req.election.election_id}`)
```

Comment:
can we start using Logger.info from now on?

Reply:
Merged and added Logger